build.sbt

```diff
@@ -21,4 +21,6 @@ lazy val intellijHaskell = (project in file(".")).
     unmanagedJars in Compile += baseDirectory.value / "idea"
   )
 
-ideaBuild in ThisBuild := "163.7743.44"
\ No newline at end of file
+ideaBuild in ThisBuild := "163.7743.44"
+
+javacOptions ++= Seq("-encoding", "UTF-8")
```

Comment:
@rikvdkleij Nope, I currently work with OSX 10.10.5 and I got all my encoding in UTF-8.

![image](https://cloud.githubusercontent.com/assets/6234553/21588056/bc2dcc36-d11d-11e6-9797-95d2df3a9d3c.png)